### PR TITLE
Issue 323

### PR DESCRIPTION
**Pull request by jofeinstein**
Changed if statement in `ProjectDetailView` `get_context_data()` so that superusers do not miss block of code starting at line 148.

Changed ordering of buttons on Project Detail page so that both warning buttons are on the right.

Added `test_remove_self_superuser ` to test if superusers can conditionally see "Leave Project" button and request to leave the project.

### CODE REVIEW

File: coldfront/core/project/templates/project/project_detail.html [hunk 1]
```diff
@@ -218,52 +218,51 @@ <h2 class="d-inline">
...
-                  <br><b>Project:</b> {{ project.name }}
-                </div>
-
-                <div class="modal-footer">
-                  <button type="button" class="btn btn-secondary" data-dismiss="modal">Close</button>
-                  <button class="btn btn-danger mr-1" type="submit">
-                    Yes, leave this project.
-                  </button>
-                </div>
+              <a class="btn btn-success" href="{% url 'project-add-users-search' project.pk %}" role="button"><i class="fas fa-user-plus" aria-hidden="true"></i> Add Users</a>
```
Comment (matthew-li) [inline coldfront/core/project/templates/project/project_detail.html:226]: Minor question: Is your editor somehow converting spaces to tabs and modifying the indent level for HTML?

It makes the diffs bigger than they actually are.

**Review by matthew-li — CHANGES_REQUESTED** [review]
Thanks for addressing this. I had one comment/question about changing HTML indentation.

File: coldfront/core/project/templates/project/project_detail.html (hunk 1 above)
Comment (jofeinstein) [reply coldfront/core/project/templates/project/project_detail.html:226]: Yea, I think its weird with html formatting. I'm using pycharm and don't think I changed any settings, but ill look into it.

Comment (matthew-li) [reply coldfront/core/project/templates/project/project_detail.html]: I've resolved the over-indents--let me know if the diff reflects what you changed.

In PyCharm, for an HTML file, in the bottom-right, there should be "K spaces" --> "Configure Indents for HTML...". Does yours differ from this?

pycharm_settings

Comment (jofeinstein) [reply coldfront/core/project/templates/project/project_detail.html]: It was different in my editor. I changed it to match yours.

**Review by matthew-li — APPROVED** [review]

**Pull request merged**